src/marqo/README.md

```diff
@@ -23,29 +23,24 @@ git clone https://github.com/marqo-ai/marqo.git
 cd marqo
 pip install -r requirements.txt
 ```
-
-4. `cd` into `src/marqo/tensor_search`
-5. Run the following command:
+4. Run the following command:
 ```bash
 # if you are running Marqo-OS locally: 
-export OPENSEARCH_URL="https://localhost:9200" && 
-    export PYTHONPATH="${PYTHONPATH}:<YOUR ABSOLUTE PATH TO>/marqo/src" &&
-    uvicorn api:app --host 0.0.0.0 --port 8882 --reload
+export OPENSEARCH_URL="https://localhost:9200" 
+export PYTHONPATH="${PYTHONPATH}:$(pwd)/src"
```

Comment:
user must ensure they are in the marqo directory here, or else $(pwd) will be wrong

Reply:
This instruction exists above: https://github.com/marqo-ai/marqo/blob/b140f4384c3f3d29dac6c69f20de6ac6561eb109/src/marqo/README.md?plain=1#L23